eng/pipelines/stress/http-linux.yml

```
@@ -20,11 +20,12 @@ steps:
 
 - bash: |
     cd '$(Build.SourcesDirectory)/$(HttpStressProject)'
-    docker build -t $(httpStressImage) --build-arg SDK_BASE_IMAGE=$(sdkBaseImage) --build-arg CONFIGURATION=$(BUILD_CONFIGURATION) .
+    docker build -t $(HhttpStressImage) --build-arg SDK_BASE_IMAGE=$(sdkBaseImage) --build-arg CONFIGURATION=$(BUILD_CONFIGURATION) .
```

This looks like a copy/paste error.

```suggestion
    docker build -t $(httpStressImage) --build-arg SDK_BASE_IMAGE=$(sdkBaseImage) --build-arg CONFIGURATION=$(BUILD_CONFIGURATION) .
```

---

Good catch, thanks!